WordPress/Classes/Utility/App Configuration/AppConfiguration.swift

```diff
@@ -13,4 +13,5 @@ import Foundation
     @objc static let showsReader: Bool = true
     @objc static let showsCreateButton: Bool = true
     @objc static let showsQuickActions: Bool = true
+    @objc static let showsFollowedSites: Bool = true
```

Comment:
```suggestion
    @objc static let showsFollowedSitesSettings: Bool = true
```

Wdyt about this change?

Reply:
Fixed in 024b936